protos/foundation/protos/doc/entity.proto

```diff
@@ -36,16 +38,20 @@ message GenericEntity {
 	// The 'type' field tells a decoder how to deserialize the 'data' field.
 	required string type = 1;
 	required bytes data = 2;
+	required geometry.BBox bbox = 3;
```

Comment:
I think we should make a geometry.BBoxes type that is simply:

```
message BBoxes {
    repeated BBox bounding_boxes = 1;
}
```

Reply:
Ok it sounds like we have two choices:

1. Every entity has a `BBox`.
2. Every entity has `BBoxes`. You can compute `entity.bbox` by unioning its `bboxes`.

I prefer (1). It is simpler. Composite entities can have references to their child entities in cases where that is required, and those child entities will by definition have bboxes.

What is the value of (2)? If we just have a list of bboxes in an Entity, are those by convention associated with some list of other stuff in the entity (like raw strings corresponding to words)? If we want to keep a sublist of words, why not keep a list of word entities (which are string + bbox)?